docs/arcaflow/workflows/flow-control.md

```diff
@@ -243,13 +257,15 @@ outputs:
 
 #### Alternative methods
 
-For handling disabled steps, `!oneof` and `!ordisabled` are the recommended methods because they cause output failure
-when the step fails. However, if it is acceptable for the workflow to succeed when a step crashes, the optional tags
-could be the right feature for your workflow.
+For handling disabled steps, `!oneof` and `!ordisabled` are the recommended methods
+because they cause output failure when the step fails. However, if it is acceptable for
+the workflow to succeed when a step crashes, the optional tags could be the right
+feature for your workflow.
 
-The alterative methods are the "optional" family of expression tags. The `oneof` tags instructed the workflow to
-build a `oneof_string` type with OR dependencies, requiring one of the options to have an output for the oneof to
-resolve. Meanwhile, the "optional" family of tags can resolve without an output by utilizing optional object property
+The alterative methods are the "optional" family of expression tags. The `oneof` tags
```

Comment:
You introduce the concept of "optional" tags in the first paragraph, and then introduce the concept again in the following paragraph, which sounds repetitive. Maybe the final phrase of the first paragraph should be "there are alternative methods." Or else drop "The alternative methods are" from the follow-up and just start explaining the "optional" family. (And "optional" family seems better than "optional tags", which suggests the English concept "optional" rather than the keywords representing optional dependencies in the workflow. 😆 )